drivers/espi/CMakeLists.txt

```diff
@@ -10,8 +10,8 @@ zephyr_library_sources_ifdef(CONFIG_ESPI_NPCX		host_subs_npcx.c)
 zephyr_library_sources_ifdef(CONFIG_ESPI_TAF_NPCX	espi_taf_npcx.c)
 zephyr_library_sources_ifdef(CONFIG_USERSPACE		espi_handlers.c)
 zephyr_library_sources_ifdef(CONFIG_ESPI_EMUL		espi_emul.c)
-zephyr_library_sources_ifdef(CONFIG_ESPI_SAF_XEC	espi_saf_mchp_xec.c)
```

Comment:
Is is possible to change the source file name from espi_saf_mchp_xec.c to espi_taf_mchp_xec.c
Same for espi_saf_mchp_xec_v2.c

Reply:
That can be done as a separate commit or separate PR.

Reply:
ok